### ISSUE TITLE
JBI blows away any non-bugzilla labels set on issues in Jira

### ISSUE THREAD

**Issue by bsieber-mozilla** [mozilla/jira-bugzilla-integration#201]
_This seems to be a concern to a stakeholder, capturing this as an issue to discuss and prioritize._

<img width="905" alt="Screen Shot 2022-08-11 at 11 04 31 AM" src="https://user-images.githubusercontent.com/53873986/184208448-32c1249d-d862-417a-902b-a1776baba12b.png">

Slack ref: https://mozilla.slack.com/archives/C016BTJKM9B/p1660239831970749

**Comment by Mossop** [comment]
In the MR2 Jira project we use labels to track whether issues have been triaged and the status of QA work for the issue. Now that we're syncing from Bugzilla we're losing those status labels.

**Comment by Mossop** [comment]
Would a change that added a config option to not sync the labels, or to sync them to a different Jira field be accepted?

**Comment by bsieber-mozilla** [comment]
Yeah, if it's a parameter like config_map and used in the custom action; feel free to modify the logic

**Comment by Mossop** [comment]
In this case it would need to be changed for the default action but yeah.

**Comment by Mossop** [comment]
This has been fixed by #206
